wrangler-service/src/main/java/io/cdap/wrangler/service/directive/RemoteExecutionTask.java

```diff
@@ -107,7 +107,7 @@ public void run(RunnableTaskContext runnableTaskContext) throws Exception {
                                                                           namespace,
                                                                           ExecutorContext.Environment.SERVICE,
                                                                           systemAppContext,
-                                                                          new DefaultTransientStore()))) {
+                                                                          new DefaultTransientStore()), null)) {
```

Comment:
Might be a good idea to set it to a default Hashmap everywhere instead of null where each column probably maps to NO_ACTION. I think it would be easier in later implementations.